roles/activemq/meta/argument_specs.yml

```diff
@@ -5,5 +5,11 @@ argument_specs:
     author: Alfresco
     options:
       activemq_version:
-        description: activemq version to use for deployment (URL still handled via group_vars)
+        description: activemq version to use for deployment
         required: true
+      activemq_dependency_url:
+        description: activemq dependency URL to use for deployment
+        required: false
+      activemq_dependency_sha512_checksum_url:
+        description: activemq dependency sha512 checksum URL to use for deployment
+        required: false
```

Comment:
I'd avoid reusing the playbook var names and use a more generic one within the role: `activemq_url` & `activemq_checksum`.
From the role point of view activemq is not a dependency and the checksum could be any other type of hash